.flowconfig

```diff
@@ -1,7 +1,7 @@
 [ignore]
 .*/src/api/.*
 .*/src/core/.*
-.*/dist/.*
+.*/(?<!ripple-keypairs/)dist/.*
```

Comment:
Yeah, I wasn't sure if it was really working. I'll just update ripple-keypairs to take `random` in the generateWallet, and use some different folder instead of dist, and get on with life.